python/private/repo_utils.bzl

```diff
@@ -66,18 +67,25 @@ def _logger(rctx):
         "TRACE": 3,
     }.get(verbosity_level, 0)
 
+    if hasattr(ctx, "attr"):
+        # This is `repository_ctx`.
+        name = name or "{}(@@{})".format(getattr(ctx.attr, "_rule_name", "?"), ctx.name)
+    elif not name:
+        fail("The name has to be specified when using the logger with `module_ctx`")
+
+    name = getattr(ctx, "name", "extension")
```

Comment:
This assignment is clobbering the logic above. I think you meant for this to be in the elif branch?

Reply:
Oops, this was leftover code, now removed.